static/app/views/issueList/customViewsHeader.tsx

```diff
@@ -162,15 +162,6 @@ function CustomViewsIssueListHeaderTabsContent({
   );
 
   const getInitialTabKey = () => {
-    if (draggableTabs[0].key.startsWith('default')) {
-      if (query) {
-        return TEMPORARY_TAB_KEY;
-      }
-      return draggableTabs[0].key;
-    }
-    if (!query && !sort && !viewId) {
-      return draggableTabs[0].key;
-    }
```

Comment:
Hopefully this simplifies the initial tab key logic a bit. Now the logic is:

1. If the query params contain a `viewId` **and** that it matches the id of one of the user's tabs, then that tab is the initial tab no matter what.
2. If the query params contain a `query`, then the initial tab is a temporary tab
3. For all other scenarios, the first tab is the initial tab.

Reply:
Nice! Removing lines to fix bugs, nothing better than that